src/declarations/stencil-public-compiler.ts

```diff
@@ -1975,6 +1975,22 @@ export interface OutputTargetBaseNext {
   dir?: string;
 }
 
+/**
+ * This enum defines all behavioral options available for
+ * exporting elements from the `dist-custom-elements` output target.
+ */
+export enum CustomElementsExportBehavior {
+  /**
+   * No additional export or definition behavior will happen.
+   */
+  DEFAULT = 'default',
+  /**
+   * All components will be re-exported from the specified directory's
+   * root `index.js` file.
```

Comment:
That's not entirely true tho since treeshaking does work as long as `autoDefineCustomElements` isn't enabled. Which, by the time V3 is released, those two behaviors will not be able to be used together.